### PR TITLE
Allow typed list + objects in OMG arguments

### PR DESCRIPTION
**Pull request by wilzbach**
This should help bringing OMG and Storyscript closer together. I tried to model them after the existing OMG style. Feedback welcome.

### CODE REVIEW

File: schema/actions/README.md [hunk 1]
```diff
@@ -188,6 +188,90 @@ actions:
...
+            type: int
+```
+
+This service argument must be a `map` with `string` as keys and integers as value.
+
+### Objects
+
+<Badge text="actions.$.arguments.$.objects" type="tip"/> <Badge text="actions.$.events.$.arguments.$.objects" type="tip"/>
+
+The argument `type` can also be an object.
```
Comment (wilzbach) [inline schema/actions/README.md]: This is now allowed as of https://github.com/microservices/microservice.guide/pull/109, but imho we should still be explicit here.

File: schema/actions/README.md [hunk 2]
```diff
@@ -188,6 +188,90 @@ actions:
...
+
+The optional argument `list` must have a key `element` with the type of its list elements.
+
+```yaml{6}
+actions:
+  colorize:
+    arguments:
+      colors:
+        type: list
+        list:
```
Comment (judepereira) [inline schema/actions/README.md]:
```suggestion
        items:
```

File: schema/actions/README.md [hunk 3]
```diff
@@ -188,6 +188,90 @@ actions:
...
+          element:
```
Comment (judepereira) [inline schema/actions/README.md]:
```suggestion
          elements:
```

File: schema/actions/README.md [hunk 4]
```diff
@@ -188,6 +188,90 @@ actions:
...
+The argument `map` must have the keys `key` and `value` with the respective types of the map.
+
+```yaml{6}
+actions:
+  colorize:
+    arguments:
+      colorMapping:
+        type: map
+        map:
+          key:
```
Comment (judepereira) [inline schema/actions/README.md]:
```suggestion
          keys:
```

**Pull request merged**